src/packages/cli/src/Studio.ts

```diff
@@ -153,11 +153,14 @@ export class Studio implements Command {
 
     const serverUrl = `http://localhost:${port}`
     if (!browser || browser.toLowerCase() !== 'none') {
-      await open(serverUrl, {
-        app: browser,
-        url: true,
-        // eslint-disable-next-line @typescript-eslint/no-empty-function
-      }).catch(() => { }) // Ignore any errors
```

Comment:
Don't know what I was thinking when I added a `.catch`, it cannot swallow errors that way